recipes/corrade/all/conanfile.py

```diff
@@ -136,8 +148,10 @@ def package_info(self):
         if self.settings.os == "Linux":
             self.cpp_info.system_libs = ["m", "dl"]
 
-        self.cpp_info.builddirs = [os.path.join(self.package_folder, "lib", "cmake", "Corrade")]
-
-        bindir = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH environment variable: {}".format(bindir))
-        self.env_info.PATH.append(bindir)
+        self.env_info.PATH.append(os.path.join(self.package_folder, "bin"))
+        if self.options.shared:
+            # Help linker find rpaths
+            if self.settings.os == "Linux":
+                self.env_info.LD_LIBRARY_PATH.append(os.path.join(self.package_folder, "lib"))
+            if self.settings.os == "Macos":
+                self.env_info.DYLD_LIBRARY_PATH.append(os.path.join(self.package_folder, "lib"))
```

Comment:
```suggestion
```

Conan already manipulates it: https://docs.conan.io/en/latest/devtools/create_installer_packages.html#using-the-tool-packages-in-other-recipes

Reply:
Pretty sure I had to add this at some point but can't find the exact failing build... But ok.